eslint.config.js

```diff
@@ -0,0 +1,41 @@
+import globals from 'globals';
+import js from '@eslint/js';
+import pluginReact from 'eslint-plugin-react';
+
```

Comment:
I highly suggest to add [eslint-config-prettier](https://github.com/prettier/eslint-config-prettier) cause prettier could conflict with linters as it is [described](https://prettier.io/docs/integrating-with-linters) in an official documentation.

Here is one of the possible [examples](https://github.com/Code-the-Dream-School/ii-practicum-team-2-back/blob/main/eslint.config.mjs#L5) from our BE repo.